src/ralph/static/src/scss/_settings.scss

```diff
@@ -38,10 +39,20 @@ $button-margin-bottom: 0;
 $button-radius: 0;
 
 $form-label-font-size: rem-calc(12);
+
 $fieldset-border-style: solid;
-$fieldset-border-width: 1px;
-$fieldset-border-color: #ccc;
-$fieldset-padding: rem-calc(30);
+$fieldset-border-width: 3px;
+$fieldset-border-color: #eee;
+$fieldset-padding: 10px 10px 10px 10px !important;
```

Comment:
True story bro, so please give me hint here how to overrride this particular behavior :-)

There's no more specific selector which could override all fields across the application.
We can patch foundation sources, but it could be even worse solution. 

So, ideas welcome!